internal/services/network/network_watcher_flow_log_resource.go

```diff
@@ -409,13 +439,10 @@ func resourceNetworkWatcherFlowLogDelete(d *pluginsdk.ResourceData, meta interfa
 		return fmt.Errorf("retreiving %s: `properties` or `properties.TargetResourceID` was nil", id)
```

Comment:
Would you mind fixing this teeny typo here
```suggestion
		return fmt.Errorf("retrieving %s: `properties` or `properties.TargetResourceID` was nil", id)
```